packages/matrix/tests/registration-with-token.spec.ts

```diff
@@ -288,92 +288,8 @@ test.describe('User Registration w/ Token', () => {
     await page.locator('[data-test-name-field]').fill('user1');
     await page.locator('[data-test-email-field]').fill('user1@example.com');
     await page.locator('[data-test-username-field]').fill('user1');
-    await page.locator('[data-test-password-field]').fill('mypassword1');
-    await page
-      .locator('[data-test-confirm-password-field]')
-      .fill('mypassword1');
-    await expect(
-      page.locator(
-        '[data-test-password-field][data-test-boxel-input-validation-state="invalid"]',
-      ),
-    ).toHaveCount(1);
-    await expect(
-      page.locator(
-        '[data-test-password-field] ~ [data-test-boxel-input-error-message]',
-      ),
-    ).toHaveText(
-      'Password must be at least 8 characters long and include a number and a symbol',
-    );
-
-    await page.locator('[data-test-password-field]').fill('mypassword!');
-    await page
-      .locator('[data-test-confirm-password-field]')
-      .fill('mypassword!');
-    await expect(
-      page.locator(
-        '[data-test-password-field][data-test-boxel-input-validation-state="invalid"]',
-      ),
-    ).toHaveCount(1);
-    await expect(
-      page.locator(
-        '[data-test-password-field] ~ [data-test-boxel-input-error-message]',
-      ),
-    ).toHaveText(
-      'Password must be at least 8 characters long and include a number and a symbol',
-    );
-
-    await page.locator('[data-test-password-field]').fill('mypassword!1');
-    await page
-      .locator('[data-test-confirm-password-field]')
-      .fill('mypassword!1');
-    await expect(
-      page.locator(
-        '[data-test-password-field][data-test-boxel-input-validation-state="invalid"]',
-      ),
-      'password field does not have error state',
-    ).toHaveCount(0);
-    await expect(
-      page.locator(
-        '[data-test-password-field] ~ [data-test-boxel-input-error-message]',
-      ),
-      'password error message does not appear',
-    ).toHaveCount(0);
-
-    await page.locator('[data-test-register-btn]').click();
-    await expect(page.locator('[data-test-token-field]')).toHaveCount(1);
-  });
-
-  test(`it shows an error when password doesn't meet the requirement`, async ({
```

Comment:
This test appeared to me to be a duplicate of the above? But maybe I missed something 🤔

Reply:
looks identical to me apart from the test name, copying and searching for the entire body matches.